app/models/offer.rb

```diff
@@ -61,6 +61,14 @@ def set_state
     self.state ||= 'draft'
   end
 
+  def estimate_display
+    if high_estimate_cents.present? && low_estimate_cents.present?
+      "#{low_estimate_display} - #{high_estimate_display}"
+    else
+      low_estimate_display
+    end
+  end
+
```

Comment:
I'd like to see a unit test for this if you don't mind!

Reply:
next PR!

Reply:
There is already a helper. I'll replace this in the next PR.